### DISCUSSION TITLE
General talks

### DISCUSSION THREAD

**Discussion by mohammedawez** [Jay-study-nildana/FrontEndForStudents#43] [General]
Hlo sir ,how are you, hope everything is good , i want to tell you that at last I have came at end of  course full stack 😎...and
 also in discord i m not able to send message's add me in your friends list...

Soon after the course certificate come's I will contact you ....i m waiting...!!! @Jay-study-nildana 

@Jay-study-nildana

**Comment by Jay-study-nildana** [comment]
@mohammedawez 

1. Discord I stopped using awez. Too many students chatting with me. it was affecting my work :) If you want to stay in touch, start using email. You can email me at this email address: jay@thechalakas.com or jay@amiay.in 
1. Very good. Very good. You are at the end of the course. very nice. 

You come down to Bangalore, please come and visit me. I have a new company now, and you can see our company also. We don't have any remote work for you, but, if you (or any friends) in Bangalore are looking for work, as them to email me.

Thanks Awez

All the best, and don't forget to email me regularly

See you soon.

Jay
